src/devices/Ads1115/samples/README.md

```diff
@@ -0,0 +1,56 @@
+# Example of ADS1115
+
+## Hardware Required
+* ADS1115
+* Rotary Potentiometer
+* Male/Female Jumper Wires
+
+## Circuit
+![](ADS1115_circuit_bb.png)
+
+ADS1115
+* ADDR - GND
+* SCL - SCL
+* SDA - SDA
+* VCC - 5V
+* GND - GND
+* A0 - Rotary Potentiometer Pin 2
+
+Rotary Potentiometer
+* Pin 1 - 5V
+* Pin 2 - ADS1115 Pin A0
+* Pin 3 - GND
+
+## Code
+* First, you need to create a ADS1115 object. After that you should call Initialize() to initialize.
+    ```C#
+    // the program runs in Linux
+    // set I2C bus ID: 1
+    // ADS1115 Addr Pin connect to GND
+    // measure the voltage AIN0
+    // set the maximum range to 6.144V
+    Ads1115 adc = new Ads1115(OSPlatform.Linux, 1, AddressSetting.GND, InputMultiplexeConfig.AIN0, PgaConfig.FS6144);
+    adc.Initialize();
```

Comment:
similar comments to what I added in other PRs (no initialize, simplify sample etc) - other than that looks good